src/main/charts/jira/templates/statefulset.yaml

```
@@ -22,6 +22,9 @@ spec:
         {{- end }}
       labels:
         {{- include "jira.selectorLabels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
```

```suggestion
        {{- with .Values.additionalLabels }}
```
Will it fit your use case if we re-use `additionalLabels` here instead of creating new `podLabels`? Introducing another field for label seems a bit duplicated to me.

---

And do you mind moving it into named templates? A named template will give more flexibility to customize/override certain values. Something similar to this PR https://github.com/atlassian/data-center-helm-charts/pull/341

---

change into named templates: 5bd5bbfe97b0679f9a3236413e22ba86701ab60a

---

yup, fair enough! Thanks, Jan~